### PR TITLE
Gun rebalance

### PR DESCRIPTION
**Pull request by SpadesSlick**
## About The Pull Request

In the memory of Course.

Rebalance of most guns that were ignored by players to be something more decent.
Lowered price on all shotguns.
Gun mods should now appear much more often in loot.
Speed boost for all projectiles by 20%.

## Why It's Good For The Game

Making using some guns less of a meme and generally enhancing gunplay.

## Changelog
:cl:
balance: All projectiles are 20% faster.
balance: All shotguns now cost less.
balance: All gun discs now have lethal ammo. All shotgun ammo can be now found in gun discs. Practice ammo does not consume any license points.
balance: Vendor price for guns changed for real prices for guns. Contraband removed from FS vendor, and t's contents were moved to regular selection. There is less lethal ammo there, however.
balance: Caliber .35 received a buff, along with all non SMG guns that use it. They are now much more usable.
balance: Caliber .40 received minor buffs.
balance: Paco damage is buffed, armor penetration lowered.
balance: Lamia buffed.
balance: Mandela now have huge armor penetration capabilities so you can actually use it against protected targets.
balance: Sol is buffed substantially, lower recoil and higher damage.
balance: Decard is buffed a bit.
balance: All of the gun mods now can spawn in general technical loot. They will appear much more often.
balance: Blitzshell gun and STS are now German produced guns.
balance: STS no longer have 3 burst mode and only have 5 one, to lower the amount of mods you need to cycle though.
/:cl:

### CODE REVIEW

**Comment by jaffjafferson** [comment]
yo did you touch vendor prices?
the "price" stat on guns iirc is just for exports so you might not see any change

**Comment by SpadesSlick** [comment]
> yo did you touch vendor prices?
> the "price" stat on guns iirc is just for exports so you might not see any change

Didn't we reworked them to use item price instead?

**Comment by SpadesSlick** [comment]
No, we didn't. Okay, will fix.

File: code/modules/projectiles/guns/projectile/automatic/sts35.dm [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /obj/item/weapon/gun/projectile/automatic/sts35
-	name = "STS-35"
-	desc = "The rugged STS-35 is a durable automatic weapon, popular on frontier worlds. Uses .30 Rifle rounds. This one is unmarked."
+	name = "OR SDF \"STS-35\""
+	desc = "The rugged STS-35 is a durable automatic weapon, made by Oberth Republic Self Defence Force. \
+			Extremely efficient rifle that was put in service right before collapse of the Republic, this weapon can be found almost anywhere in the galaxy by now. \
```
Comment (Pariah919) [inline code/modules/projectiles/guns/projectile/automatic/sts35.dm:4]: Use "An extremely efficient rifle design" or "Extremely efficient rifle design". As extremely efficient rifle makes it sound like there's only one of them out there.

File: code/modules/projectiles/guns/projectile/pistol/mandella.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	name = "FS HG .25 CS \"Mandella\""
 	desc = "A rugged, robust operator handgun with inbuilt silencer. Chambered in rifle caseless ammunition, this time-tested handgun is \
 			your absolute choise if you need to take someone down silently, as it's deadly, produces no sound and leaves no traces. \
+			Build to enhance armor penetration abilities of ammo. \
```
Comment (Pariah919) [inline code/modules/projectiles/guns/projectile/pistol/mandella.dm:5]: "Built to have higher penetration capabilities than its peers." or just "Built to have enhanced armor penetration abilities." or just replace Build with Built I guess.

**Comment by SpadesSlick** [comment]
Okay, it turns out we did had a system for this, and it was never put in use. Well I fixed that.

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1045,16 +1033,15 @@
...
 					/obj/item/weapon/storage/box/shotgunammo/blanks = 50,
-					/obj/item/ammo_magazine/sllrifle = 200,
-					/obj/item/ammo_magazine/slpistol = 400,
-					/obj/item/ammo_magazine/smg/rubber = 400,
-					/obj/item/ammo_magazine/smg = 500,
-					/obj/item/ammo_magazine/ammobox/pistol = 200,
-					/obj/item/weapon/storage/box/shotgunammo/slug = 300,
-					/obj/item/weapon/storage/box/shotgunammo/buckshot = 300,
+					/obj/item/ammo_magazine/sllrifle = 400,
+					/obj/item/ammo_magazine/slpistol = 800,
```
Comment (TheShown911) [inline code/game/machinery/vending.dm:1037]: This price looks wack for a speedloader

**Comment by SpadesSlick** [comment]
Guild is being reworked now to work from prices directly, so we shall see.

**Pull request merged**